lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc

```diff
@@ -327,8 +327,8 @@ function dosomething_helpers_add_js_onboarding() {
       $onboardingSettings['slides'][] = 'InstructionSlide';
     }
 
-    if (variable_get('dosomething_campaign_onboarding_reportback_item_kudos_slide', FALSE)) {
-      $onboardingSettings['slides'][] = 'ReportbackItemKudosSlide';
+    if (variable_get('dosomething_campaign_onboarding_reportback_items_slide', FALSE)) {
```

Comment:
not needed for this PR, but a more dynamic way of doing this feels like a good idea